three-data-centers-in-two-cities-deployment.md

```diff
@@ -192,6 +192,10 @@ In the deployment of three DCs in two cities, to optimize performance, you need
     ```yaml
     config set label-property reject-leader dc 3
     ```
+   
+   > **Note:**
+   >
+   > Since TiDB 5.2, the `label-property` configuration is not supported by default. To set the replica policy, use the [Placement Rules](/configure-placement-rules.md).
```

Comment:
```suggestion
   > Since TiDB 5.2, the `label-property` configuration is not supported by default. To set the replica policy, use the [placement rules](/configure-placement-rules.md).
```